Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/PlayerLeftSessionMessage.java

```diff
@@ -6,17 +6,29 @@
 import nl.soccar.socnet.message.MessageEvent;
 
 /**
+ * Message for leaving a player of the Game.
+ *
  * @author PTS34A
  */
 @MessageEvent(id = MessageConstants.PLAYER_LEFT_SESSION_MESSAGE_ID, handler = PlayerLeftSessionMessageHandler.class)
 public final class PlayerLeftSessionMessage extends Message {
 
     private final int playerId;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)